gen/go/go.go

```diff
@@ -18,17 +18,17 @@ import (
 // go/byrelease/{c,c,c}
 // go/builder
 
-func JenniesForGo() jen.TargetJennies {
+func JenniesForGo(targetGrafanaVersion string) jen.TargetJennies {
 	tgt := jen.NewTargetJennies()
 
 	tgt.Core.Append(
-		codegen.LatestMajorsOrXJenny(filepath.Join("kinds", "core"), codegen.GoTypesJenny{}),
-		codegen.LatestMajorsOrXJenny(filepath.Join("kinds", "core"), TypedSchemaJenny{}),
+		codegen.LatestMajorsOrXJenny(filepath.Join(targetGrafanaVersion, "kinds", "core"), false, codegen.GoTypesJenny{}),
+		//codegen.LatestMajorsOrXJenny(filepath.Join(targetGrafanaVersion, "kinds", "core"), false, TypedSchemaJenny{}), // not good enough: references stuff from grafana/grafana
```

Comment:
The general solution here, to avoid the grafana/grafana dependency, is that these hopefully-reusable jennies should move into kindsys